app/Account/AccountReducer.js

```diff
@@ -119,11 +119,11 @@ export default (state = initialState, action) => {
 				resetPasswordError: true
 			};
 		}
-		case ACCOUNT_DATA_EMAIL_PREFERENCES_CHECKBOX_CHANGE: {
-			const { name, checked } = action.payload;
+		case SUBSCRIBE_TO_EMAIL_LIST: {
+			const { name } = action.payload;
 			let emailPreferences;
 			if (name === 'global') {
-				emailPreferences = { ...state.user.emailPreferences, ...{ global: checked } };
+				emailPreferences = { ...state.user.emailPreferences, ...{ global: true } };
```

Comment:
As alluded to above, do we need this action at all given that `notify_promotions` and `notify_upgrade_updates` now default to `true` in `ConfData`?

Reply:
I made this const to automatically subscribe people to sendgrid emails--the function is taken from a button in account-web that Teresa pointed me to. I believe `notify_promotions` and `notify_upgrade_updates` are different. `notify-promotions` are for displaying modals, and `notify_upgrade_updates` are for showing upgrade alerts when we push a new version. Can you double check me on this?

Reply:
You are right about `notify_promotions` and `notify_upgrade_updates`! We still may not need this action though if we now subscribe the user to emails automatically on the account server on account creation.